lib/import/Importer.js

```diff
@@ -1,5 +1,13 @@
+import {
```

Comment:
Could we also have an integration test case on `MOdeler` and `Viewer` to verify this works (i.e. take the multiple-nested-imports and see if navigation is possible)?

That helps us to ensure the feature works as intended, end-to-end. It also helps me debugging.

Reply:
👍🏻 @marstamm

Reply:
cf. https://github.com/bpmn-io/bpmn-js/pull/1590